src/screens/ModelPage/ModelPage_Phase3.js

```diff
@@ -0,0 +1,241 @@
+  MainContentInner,
+  MainContentInnerBody,
+  MapContentWrapper,
+  MapContentInner,
+  StyledNoResultsWrapper,
+  StyledNoResults,
+  SearchHeaderWrapper,
+  LoadingScreen,
+  MapWrapper,
+  CountyMapAltWrapper,
+} from './ModelPage.style';
+
+import { STATES, STATE_TO_INTERVENTION, INTERVENTIONS } from 'enums';
+import { useProjections, useStateSummary } from 'utils/model';
+
+function ModelPage_Phase3() {
+  let { stateId, countyId } = useParams();
+  // TODO(igor): don't mix uppercase and lowercase in here
+  stateId = stateId.toUpperCase();
+
+  const [mapOption, setMapOption] = useState(
+    stateId === MAP_FILTERS.DC ? MAP_FILTERS.NATIONAL : MAP_FILTERS.STATE,
+  );
+  const [mobileMenuOpen, setMobileMenuOpen] = useState(false);
+
+  let countyOption = null;
+
+  if (countyId) {
+    countyOption = _.find(
+      US_STATE_DATASET.state_county_map_dataset[stateId].county_dataset,
+      ['county_url_name', countyId],
+    );
+  }
+  const [selectedCounty, setSelectedCounty] = useState(countyOption);
+
+  useMemo(() => {
+    setSelectedCounty(countyOption);
+  }, [countyOption]);
+  const history = useHistory();
+
+  const projections = useProjections(stateId, selectedCounty);
+  const stateSummary = useStateSummary(stateId);
+
+  const stateName = STATES[stateId];
+  const intervention = STATE_TO_INTERVENTION[stateId];
+
+  const goTo = route => {
+    history.push(route);
+  };
+
+  // Projections haven't loaded yet
+  // If a new county has just been selected, we may not have projections
+  // for the new county loaded yet
+  if (!projections || projections.county !== selectedCounty) {
+    return <LoadingScreen></LoadingScreen>;
+  }
+
+  let actionTitle;
+  let actionDescription;
+  if (intervention === INTERVENTIONS.SHELTER_IN_PLACE) {
+    actionTitle = `${stateName}: Keep staying at home to protect against the COVID-19 outbreak.`;
+    actionDescription = `Avoiding hospital overload depends heavily on your cooperation.`;
+  } else {
+    actionTitle = `${stateName}: Urge your public officials to act now against the COVID-19 outbreak!`;
+    actionDescription = `To prevent hospital overload, our projections indicate a Stay at home order must be implemented soon.`;
+  }
+  let metaTags = (
+    <AppMetaTags
+      canonicalUrl={`/us/${stateId.toLowerCase()}`}
+      pageTitle={`${stateName} Forecast`}
+      pageDescription={actionTitle}
+      shareTitle={actionTitle}
+      shareDescription={actionDescription}
+    />
+  );
+
+  const renderHeader = () => {
+    return (
+      <SearchHeaderWrapper>
+        <SearchHeader
+          setMapOption={setMapOption}
+          mobileMenuOpen={mobileMenuOpen}
+          setMobileMenuOpen={setMobileMenuOpen}
+        />
+      </SearchHeaderWrapper>
+    );
+  };
+
+  const chart_modules = [
+    {
+      title: '[DEV] Projections',
+    },
+    {
+      title: '[DEV] Case Growth Rate',
+    },
+    {
+      title: '[DEV] Testing',
+    },
+  ];
+  const renderMainContent = () => {
```

Comment:
renderX is not a reacty approach to this.  these should just be React functional components